src/controllers/request.js

```diff
@@ -0,0 +1,29 @@
+import jwt from 'jsonwebtoken';
+import environment from 'dotenv';
+import models from '../db/models';
+
+environment.config();
+
+export default class usersController {
+
+  static async findAllMyRequest(req, res) {
+    const allMyRequest = await models.Request.findAll({
```

Comment:
This command seems to be repeated on line 14 of this file

Reply:
@MuhanguziDavid , That was a mistake , I have removed the first one(one outside the try and catch).
       Here is the corrected file
<img width="698" alt="Screen Shot 2020-02-20 at 15 12 48" src="https://user-images.githubusercontent.com/48021034/74937013-ea393f00-53f3-11ea-8c2c-3e3c7476743c.png">

Reply:
Cool. Ensure that your code is properly indented.